src/Uno.UI/UI/Xaml/FrameworkElement.EffectiveViewport.cs

```diff
@@ -205,11 +206,6 @@ IDisposable IFrameworkElement_EffectiveViewport.RequestViewportUpdates(bool isIn
 				return;
 			}
 
-			if (!isInitial && viewport == _parentViewport)
-			{
-				return;
-			}
-
```

Comment:
Removing this might cause invalid viewport event to be raised.

Reply:
There is another check inside `PropagateEffectiveViewportChange`, so this one was redundant (and incorrect after the changes since SCP needs to propagate the change without itself being affected by the change)

Reply:
Ok 👍(Anyway there are some tests to validate those, just so you are aware if they fail :) )